lib/aem/aemdownload.js

```diff
@@ -25,7 +25,7 @@ const { Transfer } = require("../functions/transfer");
 const { executePipeline, Pipeline } = require("../generator/pipeline");
 const { RandomFileAccess } = require("../randomfileaccess");
 const EventEmitter = require("events");
-const UploadError = require("./upload-error");
+const UploadError = require("../block/upload-error");
```

Comment:
shouldn't this be `require("../block/download-error");` ?

Reply:
Good catch. Changed.